packages/edit-site/src/components/add-template/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useState, useCallback } from '@wordpress/element';
+import { cleanForSlug } from '@wordpress/editor';
+import { __ } from '@wordpress/i18n';
+import { Modal, TextControl, Button } from '@wordpress/components';
+
+export default function AddTemplate( {
+	ids,
+	onAddTemplateId,
+	onRequestClose,
+	isOpen,
+} ) {
+	const slugs = useSelect(
+		( select ) => {
+			const { getEntityRecord } = select( 'core' );
+			return ids.reduce( ( acc, id ) => {
```

Comment:
What's `acc` here?

Reply:
"Accumulator", it's become pretty common.